### DISCUSSION TITLE
project templates

### DISCUSSION THREAD

**Discussion by sabuto** [AvaloniaCommunity/Prism.Avalonia#95] [Q&A] [answered]
Hi,

I am looking into creating some project templates for Prism.Avalonia I just want to make sure that there are none out there already that i may be missing? From what I can see there are none apart from the standard avaloniaui templates.

**Comment by DamianSuess** [comment]
@sabuto, go for it.
Though I do have a sandbox project started for one, my current efforts have been geared toward Avalonia v11 (preview) support, and then back to that.

Since this is an inquiry and not an issue, do you mind if we move this to the Discussions section?

**Comment by sabuto** [comment]
Awesome I'll have a go, not a problem moving it sorry I didnt put it in the correct place!

**Comment by sabuto** [comment, marked as answer]
Please see discussion here 

https://github.com/AvaloniaCommunity/Prism.Avalonia/discussions/37
